backend/app/models/server_side_session.rb

```diff
@@ -0,0 +1,27 @@
+class ServerSideSession < ActiveRecord::SessionStore::Session
+  belongs_to :user, optional: true
+
+  before_update :set_user_id
+
+  ACTIVE_SESSION_LIMIT = 10
+
+  scope :by_oldest, -> { order(updated_at: :asc) }
+  after_commit :limit_active_sessions
+
+  private
+
+  def limit_active_sessions
+    if ServerSideSession.where(user_id: user_id).count > ACTIVE_SESSION_LIMIT
+      ServerSideSession
+         .where(user_id: user_id)
+         .by_oldest
+         .limit(1)
```

Comment:
can you do something like this instead of calling `delete_all` on a list with one item?

```ruby
ServerSideSession
         .where(user_id: user_id)
         .by_oldest
         .first
         .delete
```